Snippets/Snippets_5/Azure/Transports/AzureServiceBus/NamingConventionsUsage.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace Snippets5.Azure.Transports.AzureServiceBus
+{
+    using System;
+    using System.Text.RegularExpressions;
+    using NServiceBus;
+    using NServiceBus.Azure.Transports.WindowsAzureServiceBus.QueueAndTopicByEndpoint;
+    using NServiceBus.Settings;
+
+    public class NamingConventionsUsage
+    {
+        public void Snippet()
+        {
+            #region ASB-NamingConventions-entity-sanitization 6.4
+
+            NamingConventions.EntitySanitizationConvention = (entityName, entityType) =>
+            {
+                if (entityType == EntityType.Queue || entityType == EntityType.Topic)
+                {
+                    var regexQueueAndTopicValidCharacters = new Regex(@"[^a-zA-Z0-9\-\._\/]");
```

Comment:
Done. @SimonCropp should this be a compilation error in the snippets project (R# enforced)?